osvr/RenderKit/DistortionParameters.cpp

```diff
@@ -35,7 +35,6 @@ namespace osvr {
     OSVR_RENDERMANAGER_EXPORT DistortionParameters::DistortionParameters(
       OSVRDisplayConfiguration& osvrParams,
       size_t eye) : DistortionParameters() {
-      m_desiredTriangles = osvrParams.getDesiredDistortionTriangleCount();
```

Comment:
Looks like an accidental revert of a change on master. Recommend merging master or rebasing.